articles/hooks/update.md

```diff
@@ -1,61 +1,93 @@
 
-You can update existing Hooks using either the Dashboard or the command-line interface.
+You can update Hooks configured for any given [extensibility point](/hooks/extensibility-points) using the Dashboard or Management API.
 
-## Update Hooks using the Dashboard
+Hooks may also be imported and exported using the [Deploy Command-Line Interface (CLI) Extension](/extensions/deploy-cli).
 
-With the Dashboard, update the code for the existing Hooks using the [Webtask Editor](https://webtask.io/docs/editor): 
+::: note
+If you added a [Hook Secret](/hooks/secrets) and want to update it, see [Update Hook Secrets](/hooks/secrets/update).
+:::
 
-1. Navigate to [the Hooks page of the Dashboard](${manage_url}/#/hooks) and find the Hook you want to edit.
-2. Click the **Pencil and Paper** icon to the right of the Hook to open the Webtask Editor.
+<div class="code-picker">
+  <div class="languages-bar">
+    <ul>
+      <li><a href="#dashboard" data-toggle="tab">Dashboard</a></li>
+      <li><a href="#mgmt-api" data-toggle="tab">Management API</a></li>
+    </ul>
+  </div>
+  <div class="tab-content">
+    <div id="dashboard" class="tab-pane active">
 
-  ![List of Hooks](/media/articles/hooks/hooks-list.png)
-
-3. Edit the Hook using the Webtask Editor.
-
-  ![Webtask Editor](/media/articles/hooks/webtask-editor.png)
-
-### Rename Hooks
+## Rename Hooks using the Dashboard
 
-To rename a Hook using the Dashboard:
-
-1. On [the Hooks page of the Dashboard](${manage_url}/#/hooks), find the Hook you want to edit.
-2. Click the **Gear** icon next to the Hook.
-3. Click **Rename**. You will see a dialog pop up, asking you for the **Current Name** of the Hook, as well as the **New Name** you want to use. Click **Rename** when you have populated both values.
+1. Navigate to the [Hooks](${manage_url}/#/hooks) page in the [Auth0 Dashboard](${manage_url}/), and click the gear icon next to the Hook you want to rename.
+2. Select **Rename**.
+3. Enter the current name and new name of the hook, then click **Rename**.
 
 ![Rename Hooks prompt](/media/articles/hooks/rename-hook.png)
 
-## Update Hooks using the command-line interface
-
-The Auth0 Command-Line Interface (CLI) allows you to update existing Hooks associated with specific extensibility points within the Auth0 platform.
+## Update Hook scripts using the Dashboard
 
-<%= include('./_includes/set-up-webtask-cli') %>
+1. Navigate to the [Hooks](${manage_url}/#/hooks) page in the [Auth0 Dashboard](${manage_url}/), and click the pencil icon next to the Hook you want to update.
 
-Edit the code of the Hook using the [Webtask Editor](https://webtask.io/docs/editor). The following command will open up the code for the Hook in the Webtask Editor window:
+  ![List of Hooks](/media/articles/hooks/hooks-list.png)
 
-```bash
-auth0 edit my-extension-1
+2. Update the Hook using the Hook Editor, and click the disk icon to save.
+
+  ![Update a Hook in the Hook Editor](/media/articles/hooks/webtask-editor.png)
```

Comment:
This screenshot is out of date, as it shows the webtask editor, not the embedded hook editor.